docs/guides/variables.md

```diff
@@ -57,38 +62,40 @@ You can access any variable through the Python SDK with the `Variable.get()` met
 ```python
 from prefect.variables import Variable
 
-# setting the variable
-variable = Variable.set(name="the_answer", value="42")
-
-# getting from a synchronous context
-answer = Variable.get('the_answer')
-print(answer.value)
-# 42
-
-# getting from an asynchronous context
-answer = await Variable.get('the_answer')
-print(answer.value)
-# 42
-
-# getting without a default value
-answer = Variable.get('not_the_answer')
-print(answer.value)
-# None
-
-# getting with a default value
-answer = Variable.get('not_the_answer', default='42')
-print(answer.value)
-# 42
-
-# using `overwrite=True`
-answer = Variable.get('the_answer')
-print(answer.value)
-#42
-answer = Variable.set(name="the_answer", value="43", overwrite=True)
-print(answer.value)
-#43
+# set a variable
+Variable.set(name="the_answer", value="42")
+
+# get a variable
+answer = Variable.get("the_answer")
+print(answer) # 42
+
+# unset a variable
+Variable.unset("the_answer")
+
+# get a variable that doesn't exist
+answer = Variable.get("not_the_answer")
+print(answer) # None
+
+# getting a variable that doesn't exist with a default value
+answer = Variable.get("not_the_answer", default="42")
+print(answer) # 42
+
+# overwrite an existing variable
+Variable.set(name="the_answer", value="43", overwrite=True)
+answer = Variable.get("the_answer")
+print(answer) #43
 ```
 
+All standard methods are also available as async methods:
```

Comment:
When should I use sync methods vs. async methods?

Reply:
good call, need to update this because sync version won't work outside of tasks/flows anymore